.github/workflows/_rust.yml

```diff
@@ -130,7 +130,7 @@ jobs:
       - name: Compile Tailwind
         run: pnpm tailwindcss -i src/input.css -o src/output.css
       - name: Build client
-        run: cargo build -p firezone-gui-client
+        run: cargo build -p firezone-gui-client --all-targets
```

Comment:
This allows us to remove `cargo build` invocations from the gui-smoke-test Rust binary